lighthouse-core/report/v2/renderer/util.js

```diff
@@ -35,15 +35,16 @@ class Util {
   /**
    * Format number.
    * @param {number} number
+   * @param {number} decimalPlaces Number of decimal places to include. Defaults to 1.
    * @return {string}
    */
-  static formatNumber(number) {
-    return number.toLocaleString(undefined, {maximumFractionDigits: 1});
+  static formatNumber(number, decimalPlaces = 1) {
+    return number.toLocaleString(undefined, {maximumFractionDigits: decimalPlaces});
   }
 
   /**
    * @param {number} size
-   * @param {number=} decimalPlaces Number of decimal places to include. Defaults to 2.
+   * @param {number} decimalPlaces Number of decimal places to include. Defaults to 2.
```

Comment:
haha, no worries! here's a [quick guide](https://github.com/google/closure-compiler/wiki/Annotating-Types) to the closure types in case you want to decipher what all these comments mean :)